docs/quick-start.rst

```diff
@@ -3,12 +3,21 @@ SNARE
 
 Super Next generation Advanced Reactive honEypot
 
+Snare is a web application honeypot and is the successor of Glastopf, which has many of the same
+features as Glastopf as well as ability to convert existing Web pages into attack surfaces with TANNER_.
+Every event sent from SNARE to TANNER_ is evaluated, and TANNER_ decides how SNARE should respond to
+the client. This allows the honeypot to produce dynamic responses which improves its camouflage.
+
+.. _TANNER: https://github.com/mushorg/tanner
+
 Basic Concepts
 """"""""""""""
 
 * Surface first. Focus on the attack surface generation.
 * Sensors and masters. Lightweight collectors (SNARE) and central decision maker (tanner).
-
+* Snare uses ``Cloner`` which clone the application, saves the files in ``/opt/`` and it serves it on top of itself
```

Comment:
Basically, I do not think it can be a "basic concept": it's a way how the 1st concept is implemented. Same for the next point. Any opinions on that?

Reply:
@afeena I agree with you. So should we merge the 3rd and 4th points with the first?

Reply:
@Aniket21mathur try to keep it short :) I'd recommend you to delete 3d point and rephrase the 4th one) or if you find a good way of representing both points, feel free to add :)

Reply:
I have made some changes. Please review :-)